modules/module.c

```diff
@@ -112,29 +116,25 @@ static int import(size_t argc, struct cb_value *argv, struct cb_value *result)
 		}
 	} else {
 		/* FIXME: Keep track of current file directory so pwd can be set */
-		f = cb_agent_resolve_import(import_name, NULL, &path);
+		f = cb_agent_resolve_import(import_name_str, NULL, &path);
 		if (!f)
 			goto err;
 	}
 
-	pc = cb_bytecode_len(cb_vm_state.bytecode);
-	if (cb_compile_module(cb_vm_state.bytecode, import_name, f, path))
+	import_name = cb_agent_intern_string(cb_strptr(&import_name_str),
+			cb_strlen(import_name_str));
+	modspec = cb_agent_get_modspec_by_name(import_name);
```

Comment:
Maybe there should be another function to find a modspec by uninterned string so that we're not interning strings at run time

Reply:
it would be interned anyway when the module is created